index.html

```diff
@@ -1,111 +1,113 @@
-                        name="recomandation" />Definitely</label>
-                <label class="form-check-label col-md-12 my-2" for="option-2"><input class="form-check-input" id="option-2" value="Maybe" type="radio"
-                        name="recomandation" />
-                    Maybe</label>
-                <label class="form-check-label col-md-12 my-2" for="option-3"><input class="form-check-input" id="option-3" value="Not sure" type="radio"
-                        name="recomandation" />Not sure</label>
-            </div>
-            <p class="mb-0">What would you like to see improved? (Check all that apply)</p>
-            <div class="form-check my-2">
-                <label class="form-check-label col-md-12 my-2" for="check-1"><input class="form-check-input" id="check-1" value="Front-end Projects"
-                        type="checkbox" />Front-end Projects</label>
-                <label class="form-check-label col-md-12 my-2" for="check-2"><input class="form-check-input" id="check-2" value="Back-end Projects"
-                        type="checkbox" />Back-end Projects</label>
-                <label class="form-check-label col-md-12 my-2" for="check-3"><input class="form-check-input" id="check-3" value="Data Visualization"
-                        type="checkbox" />Data Visualization</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Open Source Community"
-                        type="checkbox" />Open Source Community</label>
-                <label class="form-check-label col-md-12 my-2" for="check-3"><input class="form-check-input" id="check-3" value="Gitter help rooms"
-                            type="checkbox" />Gitter help rooms</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Videos"
-                            type="checkbox" />Videos</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="City Meetups"
-                                type="checkbox" />City Meetups</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Wiki"
-                                    type="checkbox" />Wiki</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Forum"
-                                        type="checkbox" />Forum</label>
-                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Additional Courses"
-                                            type="checkbox" />Additional Courses</label>
-                    
-            </div>
```

Comment:
Format your code properly to make them readable

Reply:
- [ ] I would suggest while giving a code review. It is better to explain them kindly instead of giving direct orders.


> Example: I think it would be better if you could format your code properly to make them readable.

Reply:
> 

- [ ] I would suggest while giving a code review. It is better to explain them kindly instead of giving direct orders.


> Example: I think it would be better if you could format your code properly to make them readable.